tests/acceptance/features/webUIAccount/accountInformation.feature

```diff
@@ -0,0 +1,29 @@
+Feature: View account information
+  As a user
+  I would like to view my account information
+  So that I can verify and use my account details correctly
+
+  Background:
+    Given user "user1" has been created with default attributes
+
+  Scenario: view account information when the user has been created without groups membership
+    Given user "user1" has logged in using the webUI
+    When the user browses to the account page
+    Then the user should have following details displayed on the account information
+      | Username          | user1                         |
+      | Display name      | User One                      |
+      | Email             | user1@example.org             |
+      | Groups membership | You are not part of any group |
+
+  Scenario: view account information when the user has been added to a group
```

Comment:
try also a group membership of multiple groups

Reply:
done